modules/channel-teams/src/backend/client.ts

```diff
@@ -122,17 +148,13 @@ export class TeamsClient {
       msg = this._prepareChoicePayload(event)
     }
 
-    await this.adapter.continueConversation(ref, async (turnContext: TurnContext) => {
-      try {
+    try {
+      await this.adapter.continueConversation(ref, async (turnContext: TurnContext) => {
         await turnContext.sendActivity(msg)
-      } catch (err) {
-        const message = `The following error occured when sending a payload of type ${msg.type} to MS Botframework: ${
-          err.message
-        }`
-        this.bp.logger.error(message, err)
-        throw err
-      }
-    })
+      })
+    } catch (err) {
+      this.logger.attachError(err).error(`Error while sending payload of type "${msg.type}" `)
```

Comment:
Should'nt we re-throw the error here?

Reply:
I don't think so, otherwise it will re-try to send the message, and the console gets spammed. Just printing a log indicating that there's an error seems to be enough

Reply:
:+1: